lib/bitcoin/BitcoinClient.ts

```diff
@@ -481,6 +485,9 @@ export default class BitcoinClient {
     previousFreezeTransaction: Transaction,
     previousFreezeUntilBlock: number): Promise<Transaction> {
 
+    // tslint:disable-next-line: max-line-length
+    console.info(`Creating preivously frozen amount from transaction with id: ${previousFreezeTransaction.id} which was frozen until block: ${previousFreezeUntilBlock}`);

```

Comment:
nit: The phrasing here is kind of ambiguous to me. "Creating previously frozen amount" sounds weird to me. Is it bad if we just expose the fact that we are creating a spend transaction back to the wallet specified the precious freeze transaction?

Reply:
Re: exposing the spend-to-wallet information:
We are not logging any sensitive information so I don't think that there's a risk here. The money goes back to the wallet and obviously cannot be spend w/o the private key. Please let me know if you something else.

Re: phrasing:
I am changing the log statement to be more clear.